service/controller/app/v1/resource/appcatalog/resource.go

```diff
@@ -11,62 +11,70 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/giantswarm/app-operator/service/controller/app/v1/controllercontext"
 	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
 )
 
-// Config represents the configuration used to create a new appcatalog service.
+const (
+	// Name is the identifier of the resource.
+	Name = "appcatalogv1"
+)
+
+// Config represents the configuration used to create a new appcatalog resource.
 type Config struct {
 	// Dependencies.
 	G8sClient versioned.Interface
 	Logger    micrologger.Logger
-
-	// Settings.
-	WatchNamespace string
 }
 
-// AppCatalog implements the appcatalog service.
-type AppCatalog struct {
+type Resource struct {
 	// Dependencies.
 	g8sClient versioned.Interface
 	logger    micrologger.Logger
-
-	// Settings.
-	watchNamespace string
 }
 
-// New creates a new configured appcatalog service.
-func New(config Config) (*AppCatalog, error) {
+// New creates a new configured appcatalog resource.
+func New(config Config) (*Resource, error) {
 	if config.G8sClient == nil {
 		return nil, microerror.Maskf(invalidConfigError, "%T.G8sClient must not be empty", config)
 	}
 	if config.Logger == nil {
 		return nil, microerror.Maskf(invalidConfigError, "%T.Logger must not be empty", config)
 	}
 
-	a := &AppCatalog{
+	r := &Resource{
+		// Dependencies.
 		g8sClient: config.G8sClient,
 		logger:    config.Logger,
-
-		watchNamespace: config.WatchNamespace,
 	}
 
-	return a, nil
+	return r, nil
 }
 
-// GetCatalogForApp gets the appCatalog CR specified in the provided app CR.
-func (a *AppCatalog) GetCatalogForApp(ctx context.Context, customResource v1alpha1.App) (*v1alpha1.AppCatalog, error) {
+func (*Resource) Name() string {
+	return Name
+}
+
+// getCatalogForApp gets the appCatalog CR specified in the provided app CR.
+func (r *Resource) getCatalogForApp(ctx context.Context, customResource v1alpha1.App) error {
+	cc, err := controllercontext.FromContext(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
 	catalogName := key.CatalogName(customResource)
 
-	a.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("looking for appCatalog %#q in namespace %#q", catalogName, a.watchNamespace))
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("looking for appCatalog %#q in namespace %#q", catalogName, "default"))
```

Comment:
```suggestion
	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("looking for appCatalog %#q", catalogName))
```

I think we can just drop the namespace from the logging. Since its cluster scoped I think including default is more confusing.